common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -0,0 +1,276 @@
+@elite_ranks = "gfx/interface/icons/culture_innovations/innovation_elite_ranks.dds"
+@zandalar = "gfx/interface/icons/culture_innovations/innovation_architecture.dds"
+@arkonite = "gfx/interface/icons/culture_innovations/innovation_arkonite.dds"
+@iron_star = "gfx/interface/icons/culture_innovations/innovation_iron_star.dds"
+@saronite = "gfx/interface/icons/culture_innovations/innovation_saronite_arms.dds"
+@waterspeaking = "gfx/interface/icons/culture_innovations/innovation_waterspeaking.dds"
+
+
+
+#culture_group_regional
+####################
+##Tribal##
+####################
+innovation_amphibious = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @amphibious
+	potential = {
+		OR = {
+			##Nagas
+			this = culture:naga
+			##Murlocs
+			this.culture_group = culture_group:murloc_group
+			##Jinyu
+			this.culture_group = culture_group:jinyu_group
+			##Giblin
+			this = culture:giblin
+		}
+	}
+	modifier = {
+		embarkation_cost_mult = -1
+		no_water_crossing_penalty = yes
+		movement_speed = -0.15
+		supply_duration = -0.25
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_sea_faring = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @sea_faring
+	potential = {
+		OR = {
+			##Human
+			this = culture:tirassian
+			this = culture:baradin
+			##Vrykul
+			this.culture_group = culture_group:vrykul_group
+			##Pirates
+			this = culture:pirate_culture
+			this = culture:blackwater
+			##Trolls
+			this.culture_group = culture_group:troll_group
+			##Ogres
+			this.culture_group = culture_group:ogre_group
+		}
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.25
+		embarkation_cost_mult = -0.75
+		garrison_size = -0.30
+	}
+	custom = unlocks_naval_raiding
+	custom = unlocks_sailable_major_rivers
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_lorewalking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @lorewalking
+	potential = {
+		##Pandaren
+		this.culture_group = culture_group:pandaren_group
+	}
+	modifier = {
+		monthly_lifestyle_xp_gain_mult = 0.05
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_hive_breeding = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @hive_breeding
+	potential = {
+		##Aqir
+		this.culture_group = culture_group:aqir_group
+	}
+	modifier = {
+		levy_reinforcement_rate = 0.2
+	}
+
+	flag = global_regional
+	flag = tribal_era_regular
+}
+innovation_waterspeaking = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @waterspeaking
+	potential = {
+		##Jinyu
+		this.culture_group = culture_group:jinyu_group
+	}
+	modifier = {
+		naval_movement_speed_mult = 0.1
+		caster_damage_mult = 0.1
+		caster_toughness_mult = 0.1
+		caster_pursuit_mult = 0.05
+		caster_screen_mult = 0.05
```

Comment:
The casters don't exist on this branch.
You should move these innovations to a separate branch based on the new unit branch so the casters are in.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/104783439-76d4b480-579f-11eb-8cae-8c5fccea7bc8.png)

</details>

Reply:
You haven't moved them at all.